### PR TITLE
All reminder list screen

### PR DESCRIPTION
**Pull request by swapnil-musale**
All Reminder Screen UI Changes
Screenshot_20220918_233126

### CODE REVIEW

File: app/src/main/java/com/hadiyarajesh/notex/ui/reminders/RemindersScreen.kt [hunk 1]
```diff
@@ -1,33 +1,206 @@
...
         Column(
             modifier = Modifier
                 .fillMaxSize()
                 .padding(innerPadding),
-            horizontalAlignment = Alignment.CenterHorizontally,
-            verticalArrangement = Arrangement.Center
+            horizontalAlignment = Alignment.CenterHorizontally
         ) {
-            Text(text = stringResource(id = R.string.welcome_message))
+            if (reminders.itemCount > 0) {
```
Comment (hadiyarajesh) [inline app/src/main/java/com/hadiyarajesh/notex/ui/reminders/RemindersScreen.kt:51]: Reminders are paging data. You don't need to handle item count here as i've already handled it in `private fun AllRemindersView()` as below

```kotlin
loadState.refresh is LoadState.NotLoading && loadState.append.endOfPaginationReached && reminders.itemCount < 1 -> {
                    item {
                        EmptyView(
                            modifier = Modifier
                                .fillParentMaxSize()
                                .padding(8.dp),
                            text = stringResource(id = R.string.empty_message),
                        )
                    }
                }
```

File: app/src/main/java/com/hadiyarajesh/notex/utility/ComposeSingleClickable.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import androidx.compose.foundation.LocalIndication
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.platform.debugInspectorInfo
+import androidx.compose.ui.semantics.Role
+
+internal interface MultipleEventsCutter {
```
Comment (hadiyarajesh) [inline app/src/main/java/com/hadiyarajesh/notex/utility/ComposeSingleClickable.kt:12]: This is for which functionality?

**Review by hadiyarajesh — CHANGES_REQUESTED** [review]
Kindly rebase your branch with @siddupatil3675 and then PR

**Comment by hadiyarajesh** [comment]
Also, remove those edit and delete button from main screen. move date to bottom right corner (with some spacing)

File: app/src/main/java/com/hadiyarajesh/notex/utility/ComposeSingleClickable.kt (hunk 2 above)
Comment (hadiyarajesh) [reply app/src/main/java/com/hadiyarajesh/notex/utility/ComposeSingleClickable.kt:12]: and where will we use it?

Comment (swapnil-musale) [reply app/src/main/java/com/hadiyarajesh/notex/utility/ComposeSingleClickable.kt:12]: For now I have used when user click on the reminder item.

**Comment by swapnil-musale** [comment]
> Kindly rebase your branch with @siddupatil3675 and then PR

Done

**Pull request merged**